aea/aea.py

```diff
@@ -137,8 +136,6 @@ def setup(self) -> None:
 
         :return: None
         """
-        if not self.is_programmatic:
-            self.resources.load(self.context)
```

Comment:
we don't have anymore "programmatic" usage, it is always programmatic, and require that resources are already populated.